CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ Prefix your items with `(Template)` if the change is about the template and not
 - Remove UWP references in Diagnostics.md
 - Updated `Refit` package to 8.0.0 to address security advisory.
 - Updated `MallardMessageHandlers` package to 2.0.0.
+- Added conventional commit validation stage `stage-build.yml`
```

Comment:
Since this repository is a project template, adding the commit validation should be seen as a new feature of that template. I suggest you change the conventional commit to `feat:` to reflect that (and update the PR template too). You'll also have to move the changelog note to a new header (3.6.X).